Add sig dir (#164)
canton-foundation/canton-dev-fund: sig-directory.md

# Canton Special Interest Groups (SIGs)

> # Canton Special Interest Groups (SIGs)

Author: ```suggestion
```

# Canton Special Interest Groups (SIGs)

Special Interest Groups (SIGs) are groups of architects, software engineers, and domain experts focused on specific technical or domain areas within the Canton ecosystem.

SIGs help coordinate expertise across the ecosystem and provide guidance on technical priorities, architecture, and grant proposals.

---

# Role of SIGs

> SIGs help support the Canton ecosystem by:

Reviewer: ```suggestion
SIGs work in collaboration with the Core Contributors group and the Voting group of the Technology & Operations committee of the Canton Foundation. 

SIGs help support the Canton ecosystem by:
```

• Helping the **Tech & Ops Committee Core Contributors and Voting groups** evaluate and champion grant proposals  
• Recommending architectural vision and technical priorities  
• Developing knowledge and best practices in their respective areas  

---

# How SIGs Operate

SIGs currently operate in an **ad-hoc structure** and focus on three primary activities.

### 1. Slack Discussions
Each SIG will have a dedicated Slack channel. These channels allow members to discuss technical topics, ecosystem developments, and proposals relevant to their area.

### 2. Grant Proposal Support
SIG members may:

• Review Development Fund proposals  
• Provide feedback to improve proposals  
• Act as **champions** helping applicants prepare stronger proposals  

The Tech & Ops committee encourages grant proposers to seek a **champion** with experience building within the Canton ecosystem.

### 3. Architecture and Roadmap Guidance
SIGs may also support the ecosystem by:

• Publishing problem statements  
• Advising on Requests for Proposals (RFPs)  
• Contributing to long-term architecture and roadmap discussions  

---

# Contributing to SIGs

> This document is intended to bootstrap the SIG structure by capturing experts across the Canton ecosystem.

Reviewer: ```suggestion
This document is intended to bootstrap the SIG structure by creating a record of knowledgeable technical contributors across the Canton ecosystem who are interested in sharing and developing their understanding of Canton, and helping advise on grant proposals. 
```

If you would like to:

• Join an existing SIG  
• Add additional experts  
• Propose a new SIG topic  

Please **submit a Pull Request updating this file**.

> If your expertise does not fit an existing SIG, feel free to **add a new SIG section**.

Reviewer: ```suggestion
If your expertise does not fit an existing SIG, feel free to submit a Pull Request to **add a new SIG section**.
```
